Scripts/Proxies/TwoHandedProxyBase.cs

```diff
@@ -161,6 +182,100 @@ public virtual void Update()
 				m_RightHand.localRotation = trackedObjectInput.rightRotation.quaternion;
 			}
 		}
+
+		public void AddFeedbackRequest(FeedbackRequest request)
+		{
+			var proxyRequest = request as ProxyFeedbackRequest;
+			if (proxyRequest != null)
+			{
+				m_FeedbackRequests.Add(proxyRequest);
+				ExecuteFeedback(proxyRequest);
+			}
+		}
+
+		void ExecuteFeedback(ProxyFeedbackRequest changedRequest)
+		{
+			foreach (var proxyNode in m_Buttons)
+			{
+				foreach (var kvp in proxyNode.Value)
+				{
+					ProxyFeedbackRequest request = null;
+					foreach (var req in m_FeedbackRequests)
+					{
+						var matchChanged = req.node == changedRequest.node && req.control == changedRequest.control;
+						var matchButton = req.node == proxyNode.Key && req.control == kvp.Key;
+						var sameCaller = req.caller == changedRequest.caller;
+						var priority = request == null || req.priority >= request.priority;
+						if (matchButton && priority && (matchChanged || sameCaller))
+							request = req;
+					}
+
+					if (request == null)
+						continue;
+
+					var button = kvp.Value;
+					if (button.renderer)
+						this.SetHighlight(button.renderer.gameObject, true, duration: k_FeedbackDuration);
+
+					if (button.transform)
```

Comment:
What exactly is a button without a transform?

Reply:
It could be one that we only highlight and has no tooltip. It would also be a shame to throw null refs everywhere if an asset is deleted or for some other reason the transform reference goes missing. Should we write a warning to the console in an else block?